tools/accuracy_checker/configs/resnet-50-official.yml

```diff
@@ -0,0 +1,49 @@
+models:
+  - name: resnet-50-official
```

Comment:
hm... does it real should be named official?
historicalyl,  original resnet is caffe model if I right know

Reply:
https://github.com/KaimingHe/deep-residual-networks